src/Api/Management/Entity/OrganizationInterface.php

```diff
@@ -54,4 +54,11 @@ public function setType(string $type): void;
      * @return array
      */
     public function getTypes(): array;
+
+    /**
+     * Returns TRUE if the organization is a Hybrid org.
+     *
+     * @return bool
+     */
+    public function isHybrid(): bool;
```

Comment:
Well, technically this is a BC breaking change because whoever implemented its own implementation for this interface it has to provide an implementation for this new method as well. Until s/he does not do that, its code will fail with fatal errors. IOW, users cannot just update to the new minor release because it may break their existing code. This is against semantic versioning.

This could be solved in a non-breaking way as well: Introduce a new class (`class HybridOrganisation implements OrganizationInterface`), the object type would clearly indicate if an org is a hyrid org. The organization JSON serializer would return an instance of this new class instead of an `Organisation` if in the API response contains
```json
            {
                "name": "features.hybrid.enabled",
                "value": "true"
            },
```

(This is how developer- and company serializations are also solved, the type of the object clearly indicates the type of the legal entity.)

Reply:
Btw, as their is no "shortcut" in the PHP API client to identify whether the org is monetizaton enabled, because it could be figured out with a trivial one liner code line this, probably this method should not be added either.

Reply:
I like the first approach suggested, as it seems to me this "property" is an internal implementation detail that users of this library should not need to know, and it keeps with the way the other entities are implemented. I'll close this PR in favor of this [other one #80](https://github.com/apigee/apigee-client-php/pull/80) that implements it that way.

Reply:
I like the first approach suggested, as it seems to me this "property" is an internal implementation detail that users of this library should not need to know, and it keeps with the way the other entities are implemented. I'll close this PR in favor of this [other one #80](https://github.com/apigee/apigee-client-php/pull/80) that implements it that way.